src/server/locale/US/PAY_LATER_LONG_TERM/mutations/index.js

```diff
@@ -13,6 +13,7 @@ export default function getMutations(id, type) {
         case 'LT_NQ_MIN':
         case 'LT_NQ_MAX':
             return lt_nq[type];
+        case 'LT_NO_AMMT':
```

Comment:
Fixing a spelling typo here:
```suggestion
        case 'LT_NO_AMNT':
```
But let's actually remove the `LT_NO_AMNT` case from this block and instead create a new case with a separate return `case 'LT_NO_AMNT': return lt_no_amnt[type];` so that it can use unique mutation rules.

We can also remove `case 'LT_NQ_MIN':` and `case 'LT_NQ_MAX':` in this same file (currently lines 13 and 14) and replace them with a single `case 'LT_NQ':`.